debugger.md

```diff
@@ -1,60 +1,48 @@
-﻿#Instructions for setting up the .NET Core debugger - PREVIEW
-*Updated on  March 1st 2016 - setup works for Linux & Mac.*
-
-***Currently working within Microsoft Corp-Net only.***
-
+﻿#Instructions for setting up the .NET Core debugger - EXPERIMENTAL PREVIEW
 Thank you for taking the time to try out our bits. 
 This page gives you detailed instructions on how to get your system running. 
-It will be updated on a regulary basis to always reflect the latest behaviour. Now let's get started.​
 
 ####Your Feedback​
-
 Please place your feedback [here](https://github.com/OmniSharp/omnisharp-vscode/issues). 
 
 ####Prerequisites / Known Issues / Notes
 * X64 only
```

Comment:
I would suggest we lead with the fact that this requires a recent .NET CLI bits. Something like:
* Requires .NET Core rc3-23829 or newer. These are daily builds of .NET Core that can be downloaded from the https://www.myget.org/F/dotnet-core/api/v3/index.json nuget feed. These are really daily builds of .NET Core and associated tools. So please continue if you want to dig in and have fun playing with new bits. But don't expect to go to production with these things.